price_service/server-rust/build.rs

```diff
@@ -0,0 +1,73 @@
+use std::{
+    env,
+    path::PathBuf,
+    process::Command,
+};
+
+fn main() {
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let out_var = env::var("OUT_DIR").unwrap();
+
+    // Clone the Wormhole repository, which we need to access the protobuf definitions for Wormhole
```

Comment:
I think it's better to pull a versioned release of wormhole (instead of git) and pin the version. We can do this in another PR, but please leave a todo here.